packages/world/test/Factories.t.sol

```diff
@@ -62,10 +62,13 @@ contract FactoriesTest is Test, GasReporter {
     address worldFactoryAddress = address(new WorldFactory(coreModule));
     IWorldFactory worldFactory = IWorldFactory(worldFactoryAddress);
 
+    // User defined bytes for create2
+    bytes memory _salt0 = abi.encode(0);
```

Comment:
Maybe the salt parameters could be [fuzzed](https://book.getfoundry.sh/forge/fuzz-testing) like `account` is? 

They both need to be hardcoded when called in `testWorldFactoryGas` though (just for our gas reporter).

Reply:
I added the fuzzing test.